index.js

```diff
@@ -1496,7 +1496,7 @@
   //.
   //. The T ([thrush][]) combinator. Takes a value and a function, and returns
   //. the result of applying the function to the value. Equivalent to Haskell's
-  //. `(&)` function.
+  //. `(&)` function, which is a reverse application operator.
   //.
   //. ```javascript
   //. > S.T (42) (S.add (1))
```

Comment:
If we give S.T one argument we get back a huge blob of code. How can we return a representation of : (a -> b) -> b?

Reply:
Deciding how to represent functions as strings is [tricky][1]. [`show`][2] simply returns `String (f)` for a given function `f`, which will either produce a useful result or a daunting wall of text depending on the nature of `f`. Perhaps in the REPL it would be better to see a placeholder such as `<Function>` in all cases.


[1]: https://github.com/tc39/Function-prototype-toString-revision/blob/master/proposal.html
[2]: https://github.com/sanctuary-js/sanctuary-show#show